### DISCUSSION TITLE
JBD read out

### DISCUSSION THREAD

**Discussion by timstewart88** [mr-manuel/venus-os_dbus-serialbattery#105] [Q&A]
Hello. Can I disable bms SOC readout so Venus only uses the Victron shunt for SOC?

Also what is the best uart or rs485 to usb adapter to use with the jbd?

**Comment by mr-manuel** [comment]
No, but you can use the driver SOC calulation with the current of the SmartShunt.

https://github.com/mr-manuel/venus-os_dbus-serialbattery/blob/4b1131411c193bf7f74dfd4ac0362b1524342b44/etc/dbus-serialbattery/config.default.ini#L71-L97

https://github.com/mr-manuel/venus-os_dbus-serialbattery/blob/4b1131411c193bf7f74dfd4ac0362b1524342b44/etc/dbus-serialbattery/config.default.ini#L186-L193

**Reply by timstewart88** [reply]
Would this mean the gx device would report a different soc to the smart shunt when reading the smart shunt with the victron app?

**Reply by mr-manuel** [reply]
It could be, yes. But one both are set to 100% they should be identical.

**Comment by HBK1** [comment]
You can disable BMS control in Venus OS under DVCC:
![image](https://github.com/user-attachments/assets/8ce6aac2-3bb6-4462-94aa-c33cbbd1b67e)
Then you can select Smartshunt as Battery monitor under System setup:
![image](https://github.com/user-attachments/assets/be690968-4389-4020-a769-0bbf0b14a98e)

**Comment by blissi** [comment]
Hi,
I got my battery running a few days ago and read A LOT of things to get all the information to have a smooth system.
I also have the SmartShunt and this is now one thing where I need some infos:
In the FAQ it is recommended to have the BMS as the battery monitor. But then I would lose the SoC calculation of the SmartShunt as far as I understood. As you wrote @mr-manuel , it is currently not possible to have the SmartShunt's SoC in dbus-serialbattery. Is this a general thing that can't be implemented? Or could it be done? If it is possible and you don't plan this feature, I could give it a try and create a PR.

If not possible: how close to the SmartShunt's SoC is the SoC calculation in your driver?

Lg Steven

**Reply by mr-manuel** [reply]
Check the changelog and default config file.
